### PR TITLE
US-3.4.1: Add user wallet balance

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/wallet/components/Wallet.vue [hunk 1]
```diff
@@ -36,4 +36,31 @@
...
+    data() {
+      return {
+        walletBalance: ''
+      }
+    },
+    created() {
+      this.getRewardBalance().then(reward => this.walletBalance = Math.trunc(reward.tokenBalance));
+    },
+    methods: {
+      getRewardBalance() {
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/wallet/components/Wallet.vue]: Should be done in api file like for others

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/wallet/components/Wallet.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/wallet/WalletAPI.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export function getRewardBalance() {
+    return fetch(`/portal/rest/wallet/api/account/detailsById?id=${eXo.env.portal.userName}&type=user`, {
+        method: 'GET',
+    }).then((resp) => {
+      if(resp && resp.ok) {
+        return resp.json();
+      } else {
+            throw new Error(('Error when getting reward balance'));
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/wallet/WalletAPI.js]: usless ()

File: digital-workplace-webapps/src/main/webapp/vue-app/wallet/components/Wallet.vue [hunk 3]
```diff
@@ -36,4 +36,27 @@
...
+      }
+    },
+    created() {
+      this.getRewardBalance();
+    },
+    methods: {
+      getRewardBalance() {
+        getRewardBalance().then(
+                (data) => {
+                  this.walletBalance = Math.trunc(data.tokenBalance)
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/wallet/components/Wallet.vue]: Add ;

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/wallet/components/Wallet.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/wallet/WalletAPI.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export function getRewardBalance() {
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/wallet/WalletAPI.js]: getWalletAccount

File: digital-workplace-webapps/src/main/webapp/vue-app/wallet/WalletAPI.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    return fetch(`/portal/rest/wallet/api/account/detailsById?id=${eXo.env.portal.userName}&type=user`, {
+        method: 'GET',
+    }).then((resp) => {
+      if(resp && resp.ok) {
+        return resp.json();
+      } else {
+        return null;
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/wallet/WalletAPI.js]: throw new Error ('Error when getting connections');

**Pull request merged**